packages/circuit-ui/components/SelectorGroup/SelectorGroup.tsx

```diff
@@ -52,12 +56,80 @@ export interface SelectorGroupProps {
    * Whether the user can select multiple options.
    */
   multiple?: boolean;
+  /**
+   * Whether to layout Selectors in a row or a column. Default: 'horizontal'.
+   */
+  orientation?: 'vertical' | 'horizontal';
```

Comment:
I don't think this should be configurable with a prop. The orientation can often change between viewports (vertical on mobile, horizontal on desktop) and it's too complicated to codify this in a prop. 

Circuit UI should provide a sensible default that can be customized with the `css` prop.

Reply:
I caught up on the discussions between @robinmetral and you above. I think it's fine to change the default in v3, but I still wouldn't make it configurable.

Reply:
Good point, agree. Will make it horizontal by default (as it's the most common use case), remove this prop and make sure it can be customized with `css`.